lib/Zureg/Model.hs

```diff
@@ -79,6 +88,122 @@ data Registrant = Registrant
     , rState :: Maybe RegisterState
     } deriving (Show)
 
+instance ToNamedRecord Registrant where
+    toNamedRecord (Registrant rUuid' rInfo' rState')
+        =  HM.unions [ namedRecord [ C.pack "UUID" .= rUuid' ]
+                     , toNamedRecord rState'
+                     , toNamedRecord rInfo'
+                     ]
+
+instance ToNamedRecord RegisterState where
+    toNamedRecord registerState = namedRecord [ C.pack "State" .= registerState ]
+
+instance ToNamedRecord RegisterState => ToNamedRecord (Maybe RegisterState) where
+    toNamedRecord registerState = namedRecord [ C.pack "State" .= registerState ]
+
+instance ToNamedRecord RegisterInfo => ToNamedRecord (Maybe RegisterInfo)  where
+    toNamedRecord registerInfo = case registerInfo of
+        Just registerInfo' -> toNamedRecord registerInfo'
+        Nothing            -> namedRecord [ C.pack "Name" .= "No Registration info present"]
+
+instance ToNamedRecord RegisterInfo where
+    toNamedRecord (RegisterInfo  riName' riBadgeName' riEmail' riAffiliation' riAskMeAbout' 
+                    tiTrackInterest' riTShirt' riMentor' riProject' riRegisteredAt')  
+        = HM.unions [ namedRecord [ C.pack "Name"               .= riName' 
```

Comment:
If you use `{-# LANGUAGE OverloadedStrings #-}`, you can drop the `C.pack` here (and in many other places).